virt/virt-2-5-release-notes.adoc

```diff
@@ -0,0 +1,198 @@
+[id="virt-2-5-release-notes"]
+= {RN_BookName}
+include::modules/virt-document-attributes.adoc[]
+:context: virt-release-notes
+toc::[]
+
+== About Red Hat {VirtProductName}
+
+Red Hat {VirtProductName} {VirtVersion} is supported for use on {product-title} 4.6 clusters. {VirtProductName} enables you to bring traditional virtual machines (VMs) into {product-title} where they run alongside containers, and are managed as native Kubernetes objects.
+
+//Branding-Logo
+{VirtProductName} is represented by the image:Operator_Icon-OpenShift_Virtualization-5.png[{VirtProductName}] logo.
+
+include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
+// This line is attached to the above `virt-what-you-can-do-with-virt` module.
+// It is included here in the assembly because of the xref ban.
+You can use {VirtProductName} with either the xref:../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes] or the xref:../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[OpenShiftSDN] default Container Network Interface (CNI) network provider.
+//Commented out because for GA release we have special text that covers this info. But going forward, we will want to include the following:
+//include::modules/virt-supported-cluster-version.adoc[leveloffset=+2]
```

Comment:
We can uncomment this include now.
It might also be worthwhile removing the first sentence in L9 since it's redundant (and this module uses version attribute so won't run into maintenance issues) but let's see how it looks for now. It might not be a bad thing to repeat.